app/views/criteria/_boot.js.erb

```diff
@@ -28,4 +28,41 @@
       });
     }
   });
+  function add_level(id) {
+    var rubric_form = $('#' + id);
+    var last_level = $('#' + id + " .level").first();
+    var new_level = last_level.clone();
+
+    // use regular expressions to update level numbers
+    var regex = new RegExp(/\d+/);
+    var level_num = $('#' + id + " .level").length;
+    // replace the new level names with the new level number
+    var new_name = $(new_level)[0].childNodes[1];
```

Comment:
Use a more specific selector here, instead of just looking at child order. Are there element types/classes/ids you can look for?